src/execution_plan/ops/op_update.c

```
@@ -115,26 +122,39 @@
-	op->updates_committed = true;
-
 	return _handoff(op);
 }
 
-static OpBase *UpdateClone(const ExecutionPlan *plan, const OpBase *opBase) {
+static OpBase *UpdateClone
+(
+	const ExecutionPlan *plan,
+	const OpBase *opBase
+) {
 	ASSERT(opBase->type == OPType_UPDATE);
```

cppcheck (2.10-2)

[error] Null pointer dereference

(nullPointer)